source/Graphics.js

```diff
@@ -7,7 +7,7 @@ var Graphics = (function()
 			preserveDrawingBuffer: true
 		};
 
-		var _GL = canvas.getContext("webgl", contextOptions) || canvas.getContext("experimental-webgl", contextOptions);
+		var _GL = canvas.getContext("webgl", contextOptions) || canvas.getContext("experimental-webgl", contextOptions) || canvas.getContext("webgl2", contextOptions);
```

Comment:
Interesting that it wouldn't return a webgl context but would eventually return a webgl2 one. 

What version of Chrome was this in?